front/lib/validators/default.js

```diff
@@ -0,0 +1,7 @@
+import { number, string } from "yup"
+
+export const stringValidator = string()
```

Comment:
Is it really necessary ?

Reply:
I'll be removing this, it's also not used in fact. Thanks